dbf2csv/controllers/controllers.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+from odoo.addons.web.controllers.main import \
+    serialize_exception, content_disposition
```

Comment:
`serialize_exception` and `content_disposition` are defined in [_odoo/http.py_](https://github.com/odoo/odoo/blob/d23667262b3e0c14935f58cae793c0ed7b58bd08/odoo/http.py#L1639-L1649) [[2](https://github.com/odoo/odoo/blob/d23667262b3e0c14935f58cae793c0ed7b58bd08/odoo/http.py#L695-L732)] and not in `odoo.addons.web.controllers.main`.

👍 for use of `content_disposition`, nice helper!

Reply:
Ok, thanks